### DISCUSSION TITLE
Using Ruby 1.9+ Hash syntax in the docs

### DISCUSSION THREAD

**Discussion by janko** [jeremyevans/rodauth#272] [General]
Whenever I copy-paste the Sequel migrations from the README, I always convert them to the Ruby 1.9+ Hash syntax (without hashrockets), so that the code adheres to the modern style. I noticed that some Sequel documentation uses the newer syntax, so I was wondering if you would be open to converting the Rodauth docs to it as well. I think it would make a difference for many Ruby developers 🙂

**Comment by jeremyevans** [comment]
I converted Sequel's and Roda's documentation to the new hash syntax, even though the implementation continues to use the old syntax for the most part.  I'm fine with converting Rodauth's documentation to use the new hash syntax.
